packages/docs/src/pages/en/components/selects.md

```diff
@@ -40,7 +40,7 @@ Select fields components are used for collecting user provided information from
 
 <alert type="error">
 
-  When using objects for the **items** prop, you must associate **item-text** and **item-value** with existing properties on your objects. These values are defaulted to **text** and **value** and can be changed.
+  When using objects for the **items** prop, you must associate **item-text** and **item-value** with existing properties on your objects. These values are defaulted to **title** and **value** and can be changed.
```

Comment:
```suggestion
  When using objects for the **items** prop, you must associate **item-title** and **item-value** with existing properties on your objects. These values are defaulted to **title** and **value** and can be changed.
```

Reply:
This modification confuses me as a user since the name of the props is "item-text": https://github.com/vuetifyjs/vuetify/blob/8bb752b210d25fbebcea12cd073d2ce4986f5e12/packages/vuetify/src/components/VSelect/VSelectList.ts in src and in docs: https://github.com/vuetifyjs/vuetify/blob/8bb752b210d25fbebcea12cd073d2ce4986f5e12/packages/docs/src/examples/v-select/prop-custom-text-and-value.vue
What do you think about it?

Translated with www.DeepL.com/Translator (free version)